### PR TITLE
Make distillery compatible with OTP 26

### PR DESCRIPTION
**Pull request by codeadict**
Updates distillery to fix `console` and `remote_console` not working in OTP 26 until we can move to Elixir releases.

- [x] Fixes deprecated `System.stacktrace()` calls
- [x] Removes default erlexec user script to be `Elixir.IEx.CLI`, will default to the elixir shell
- [x] Fix message about RELEASES file not being found
- [x] comment out quickcheck and property test that fails to run due to old library
- [x] Fixes sed output serialization to account for binaries
- [x] Add CI job , it fails on some IO things but we should be ok with it. 

## Tests

You can test by going to `cd test/fixtures/ordered_app` and running `mix distillery.release`, after that:

`_build/dev/rel/ordered_app/bin/ordered_app console`

image

### CODE REVIEW

**Review by leepage87 — APPROVED** [review]

**Review by DougVonMoser — APPROVED** [review]

File: .github/workflows/ci.yml
Comment (ck-work00) [inline .github/workflows/ci.yml:1]: 💯 nice!

File: test/fixtures/umbrella_app/apps/web/config/test.exs
Comment (ck-work00) [inline test/fixtures/umbrella_app/apps/web/config/test.exs:1]: L10:
`config :logger, level: :warning`

**Review by ck-work00 — APPROVED** [review]

File: test/fixtures/umbrella_app/apps/web/config/test.exs
Comment (codeadict) [reply test/fixtures/umbrella_app/apps/web/config/test.exs:1]: Done

**Review by ericthiele — APPROVED** [review]

**Review by zkayser — APPROVED** [review]

File: test/cases/release_test.exs [hunk 1]
```diff
@@ -41,7 +41,6 @@ defmodule Distillery.Test.ReleaseTest do
 
       assert 0 == prios[:kernel]
       assert 1 == prios[:stdlib]
-      assert prios[:db_connection] > prios[:connection]
```
Comment (codeadict) [inline test/cases/release_test.exs:44]: New : db_connection versions don't use connection anymore

File: test/fixtures/umbrella_app/apps/web/config/test.exs [hunk 2]
```diff
@@ -7,4 +7,4 @@ config :web, WebWeb.Endpoint,
   server: false
 
 # Print only warnings and errors during test
-config :logger, level: :warn
+config :logger, level: :warning
```
Comment (codeadict) [inline test/fixtures/umbrella_app/apps/web/config/test.exs:10]: :warn is deprecated

File: test/fixtures/umbrella_app/apps/web/mix.exs [hunk 3]
```diff
@@ -11,7 +11,7 @@ defmodule Web.Mixfile do
       lockfile: "../../mix.lock",
       elixir: "~> 1.4",
       elixirc_paths: elixirc_paths(Mix.env),
-      compilers: [:phoenix, :gettext] ++ Mix.compilers,
```
Comment (codeadict) [inline test/fixtures/umbrella_app/apps/web/mix.exs:14]: Gettext compiler is now default in Mix.compilers

File: test/fixtures/umbrella_app/rel/vm.args [hunk 4]
```diff
@@ -1,3 +1,3 @@
--name umbrella
+-name umbrella@127.0.0.1
```
Comment (codeadict) [inline test/fixtures/umbrella_app/rel/vm.args:1]: Need long names instead of -sname in newer OTPs

**Pull request merged**